scripts/ci/batch_lhci_leak.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+set -e
+NETWORK_NAME=${NETWORK_NAME:-"crm-network"}
+CRM_DOMAIN=${CRM_DOMAIN:-"localhost"}
+DEV_PORT=${DEV_PORT:-"3000"}
+NEXT_PUBLIC_PROD_PORT=${NEXT_PUBLIC_PROD_PORT:-"3001"}
+PLAYWRIGHT_TEST_PORT=${PLAYWRIGHT_TEST_PORT:-"9323"}
```

Comment:
In the .env file we have PLAYWRIGHT_TEST_PORT=9324, are you sure this port is correct?